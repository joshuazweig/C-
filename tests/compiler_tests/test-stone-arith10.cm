int main () {
    stone s1;
    stone s2;
    stone s3;
    stone s4;

    s1 = "7";
    s2 = "8";

    s3 = s1 + s1 * s2 + s2;
    print_stone(s3);

    s3 = s1 * s1 ^ s2 * s2;
    print_stone(s3);

    s1 = "100";
    s2 = "10";
    s3 = "2";

    s4 = s1 - s2 - s3;
    print_stone(s4);
}
