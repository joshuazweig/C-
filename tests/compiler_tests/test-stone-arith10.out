71
322828856
88
